### ISSUE TITLE
Routes are returning HTTP 503

### ISSUE THREAD

**Issue by izaac** [rancher/rio#603]
**Describe the bug**
Create a service and add a router in the mesh. The endpoint works fine but the router is returning HTTP 503.

**To Reproduce**
1. Create a service `rio run -n testcontainer -p 80 sangeetha/mytestcontainer` 
2. Wait until the endpoint is available.
3. Create a route: `rio router add route1/to-tsrv-v0 to default/testcontainer`
4. Wait until the router is generated. `rio router ls`

**Expected behavior**

Router Endpoint should route the requests to the app deployed.

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
Tried both:
```
v1.14.7-gke.10
v1.16.2-k3s.1
```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.1 (7678d67d)
Rio CLI Version: v0.6.0-alpha.1 (7678d67d)
```
**Additional context**
`rio systemlog`
output:
```
Nothing relevant.
```

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by rancher-max** [comment]
Working in v0.6.0-alpha.2.
Syntax might've changed a little bit: `rio router add route1/to-srv-v1 to testcontainer:v1`
Where `v1` is the version shown with `rio ps`

To do this in a different namespace: 
`rio -n issue-ns run -n testcontainer -p 80 sangeetha/mytestcontainer`
`rio ps` --> `...   testcontainer/53265b61     ... `
`rio -n issue-ns router add route1/to-srv-v1 to testcontainer:53265b61`

Note also with this container, due to nginx, the route will give an nginx 404 because it doesn't know which backend /to-srv-v1 should go to. Feel free to experiment with other containers :)
